Bunny101APICore/Bunny101APICore/appsettings.Development.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "Logging": {

```

Comment:
Preferably put the connection string in the settings. This way you can connect to different databases between environments (When or If you guys have a remote testing server)
eg.
```
"ConnectionStrings": {
    "DefaultConnection": "Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=MessagesDB;Integrated Security=True;Connect Timeout=30;"
  },
```